server/embedly.js

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+// Facade that abstracts the embed.ly endpoint.
+// TODO use Joi to validate the url and ensure key is defined
```

Comment:
This.

Clarifying the contract between layers in the app makes it trivial to distribute the layers later. Having some type of schema validation also lets us split the problem set. Part of this contract should include not stupidly bubbling up errors all the way to the top level; DB errors should be handled at the DB, not returned to a model, which might return it upwards to a controller. Ugz.